install/jsf/docs/README.txt

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please change date string to ... (c) 2018, 2020 Oracle and/or ....

Reply:
thanks!